widgetssdk/src/main/java/com/glia/widgets/helper/ResourceProvider.java

```diff
@@ -78,6 +80,14 @@ public int convertDpToIntPixel(float dp) {
         return Math.round(convertDpToPixel(dp));
     }
 
+    @Override
+    public String getResourceKey(int stringKey) {
```

Comment:
Does it make sense to add a javadoc for the `getResourceKey(int stringKey)` function? With several examples, maybe.

Reply:
We can add it, but it is not a commonly used functionality

Reply:
I think some short doc will be useful even for internal usage. Today you remember what function does, in a year you might need time to figure it out, you know. For a new team member it might not be obvious what resource key is and why do we need it in addition to keys from string res file.